services/app-web/src/common-code/dateHelpers/calendarDate.ts

```diff
@@ -1,12 +1,21 @@
 import { dayjs } from './dayjs'
 
-// We store calendar dates (dates with no time info) in UTC for consistency.
-// This formats a date correctly like '12/02/2022' for display.
-function formatCalendarDate(date: Date | undefined): string {
-    if (!date) {
+// We store calendar dates in UTC for consistency. This formats a date like '12/02/2022'.
+// These formatted dates, should only be used for display, never saved to the database.
+function formatCalendarDate(date: Date | undefined | string, timeZone?: 'UTC' | 'America/New_York'): string {
+    const tz = timeZone || 'UTC'
```

Comment:
please just require `timeZone` prop -  make developers think about what they want. 

We can't fall back to UTC anymore, there are valid cases where this should be eastern timezone, in fact it looks like from the function usage, eastern is more likely.